.github/workflows/editorconfig.yml

```diff
@@ -39,6 +41,7 @@ jobs:
     - name: Checking EditorConfig
       if: env.PR_DIFF
       run: |
+        git checkout refs/remotes/pull/${{ github.event.pull_request.number }}/merge
```

Comment:
Please move the git checkout to a separate step so that the error is clear.

<img width="1005" alt="error" src="https://user-images.githubusercontent.com/59103226/143722046-9a55bbc2-3f0c-42a3-bcd1-d9b0e7ec8141.png">

Reply:
Doesn't really need to be in a step by itself, could just have it run last one in the same step as the other `git` commands.

Reply:
I added it in a separate step now and am too lazy to retest (this is really annoying to do). Also I think doing it in a separate step directly before has the advantage that changes later don't break this in suble way (like checking out the base)